sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-firmware/coreos-firmware-99999999.ebuild

```diff
@@ -67,6 +68,11 @@ src_unpack() {
 		git-r3_src_unpack
 	else
 		default
+		# rename directory from git snapshot tarball
+		if [[ ${#MY_COMMIT} -gt 8 ]]; then
```

Comment:
I don't understand why this condition is as it is and not, e.g., `-gt 0`

Reply:
I am not sure.
I just want to keep it as similar as possible to Gentoo.
https://gitweb.gentoo.org/repo/gentoo.git/tree/sys-kernel/linux-firmware/linux-firmware-20230625_p20230724.ebuild#n92

Reply:
The only thing that strikes me as odd is that `GIT_COMMIT` does not show up anywhere else. There's `MY_COMMIT`, but not `GIT_COMMIT`. Maybe a bug in the ebuild?

Reply:
> The only thing that strikes me as odd is that GIT_COMMIT does not show up anywhere else. There's MY_COMMIT, but not GIT_COMMIT.

Right. I was wondering about that yesterday. :shrug: